app/actions/remote/license.test.ts

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import NetworkManager from '@managers/network_manager';
+import {isMinimumServerVersion} from '@utils/helpers';
+
+import {getLicenseLoadMetric} from './license';
+
+jest.mock('@constants/device', () => ({}), {virtual: true});
+jest.mock('@database/manager', () => ({}), {virtual: true});
+
+jest.mock('@managers/network_manager', () => ({
+    getClient: jest.fn(),
+}));
+
+jest.mock('@utils/helpers', () => ({
+    isMinimumServerVersion: jest.fn(),
+}));
+
+jest.mock('@utils/log', () => ({
+    logDebug: jest.fn(),
+}));
+
+jest.mock('@utils/errors', () => ({
+    getFullErrorMessage: jest.fn(),
+}));
+
+describe('Actions.Remote.License', () => {
+    const serverUrl = 'https://server.com';
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('getLicenseLoadMetric', () => {
+        it('should return null if not licensed', async () => {
+            const result = await getLicenseLoadMetric(serverUrl, '10.8.0', false);
+            expect(result).toBeNull();
+            expect(NetworkManager.getClient).not.toHaveBeenCalled();
+        });
+
+        it('should return null if server version is less than minimum', async () => {
+            (isMinimumServerVersion as jest.Mock).mockReturnValueOnce(false);
```

Comment:
Should use jest.mocked, same applies everywhere else.

But in this case, is there a reason to mock isMinimumServerVersion? By doing this it does not really matter what you set as the argument of the action